src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -120,87 +48,10 @@
   </ol>
 
   <stache-page-anchor>
-    Manual installation for Windows
+    Additional steps for Firefox
```

Comment:
The steps for Firefox are currently incorrect. They should be
- Start serving a SKYUX SPA with `skyux serve`
- Enter "about:preferences#privacy" in the Firefox address bar.
- Near the bottom of the page, select <strong>View Certificates</strong>.
- Under <strong>Servers</strong> on the Certificate Manager, select <strong>Add Exception</strong>.
- For the <strong>Location</strong>, choose `https://localhost:8000` (or whatever port in which `serve` is currently running) choose <strong>Get Certificate</strong>, and then <strong>Confirm Security Exception</strong>

Note that this exception is specific to just this port and will have to added for each port `skyux serve` runs on.

Reply:
I updated these instructions. Let me know if you see any problems.

Reply:
Could we move this above the iOS Simulator for Mac steps?